front/lib/api/assistant/conversation.ts

```diff
@@ -1075,6 +1089,19 @@ export async function* postUserMessage(
   void logIfUserUnknown();
 }
 
+/**
+ * Can a user mention a given configuration
+ */
+function canAccessAgent(
+  agentConfiguration: LightAgentConfigurationType
+): boolean {
+  return (
+    agentConfiguration.canRead &&
+    (agentConfiguration.status === "active" ||
```

Comment:
action || !== darft means !== draft right? This seems buggy?

Reply:
not sure what you mean, but it should have been `agentConfiguration.status === 'draft'`, so buggy indeen

Reply:
We likely want a case with an assertNever here to make sure the addition of a status will be considered here?